packages/components/pie-webc-core/package.json

```diff
@@ -6,6 +6,14 @@
   "main": "dist/index.js",
   "author": "Just Eat Takeaway.com - Design System Team",
   "license": "Apache-2.0",
+  "files": [
+    "src/decorators",
+    "src/functions",
+    "src/interfaces",
+    "src/mixins",
+    "src/types",
+    "index.ts"
+  ],
```

Comment:
I want to ensure that `internals` is not packaged. For now I have added these to preserve what is currently packaged and I will create a ticket to review our files for not only this package but all components in the repo.